lib/spack/spack/test/packages.py

```diff
@@ -165,3 +165,19 @@ def test_import_namespace_container_modules(self):
         import spack.pkg.builtin.mock                   # noqa
         import spack.pkg.builtin.mock as m              # noqa
         from spack.pkg.builtin import mock              # noqa
+
+    @pytest.mark.regression('2737')
+    def test_urls_for_versions(self):
+        # Checks that a version directive without a 'url' argument
+        # specified uses the default url, or one that is specific
+        # to the closest newer version found.
```

Comment:
@alalazo The last part of this comment appears stale:

> or one that is specific to the closest newer version found

It looks like the default url is *always* used if there is not a version-specific url.

Reply:
Fixed.